kernel/timeout.c

```diff
@@ -96,10 +92,10 @@ void _add_timeout(struct _timeout *to, _timeout_func_t fn, s32_t ticks)
 
 int _abort_timeout(struct _timeout *to)
 {
-	int ret = _INACTIVE;
+	int ret = -EINVAL;
```

Comment:
As mentioned elsewhere, probably best to just eliminate the return value here or make it a boolean to indicate "was active", as that's all existing callers care about.

Reply:
The return indicates the attempt to abort the timeout failed because the timeout wasn't active (valid for abort).  I believe this is the preferred way to indicate the result of the operation.